src/megaapi_impl.cpp

```diff
@@ -1495,6 +1495,136 @@ char *MegaApiImpl::getBlockedPath()
 }
 #endif
 
+void MegaApiImpl::backupFolder(const char *localFolder, const char *backupName, MegaRequestListener *listener)
+{
+    MegaRequestPrivate *request = new MegaRequestPrivate(MegaRequest::TYPE_BACKUP_FOLDER, listener);
+    request->setNodeHandle(0);
```

Comment:
Why??

Reply:
It was probably a debug thing, in order to make it easier to notify if the response changed the handle. I'll remove it.